amethyst_animation/src/skinning/systems.rs

```diff
@@ -43,18 +43,16 @@ impl<'a> System<'a> for VertexSkinningSystem {
         self.updated.clear();
 
         global_transforms.populate_modified(
-            &mut self
-                .updated_id
-                .as_mut()
-                .expect("VertexSkinningSystem missing updated_id."),
+            &mut self.updated_id.as_mut().expect(
+                "'VertexSkinningSystem::setup' was not called before 'VertexSkinningSystem::run'",
```

Comment:
Backticks for types please. Thanks!

Reply:
Will be changed. 

I'll push all changes to the first PR I made as separate commits and then squash on completion. Hence I'll close this one in favor of #1075.